### PR TITLE
big update to scientist guide

### PR DESCRIPTION
**Pull request by rabernat**
In response to feedback from our call last month, I removed a lot of the generic stuff from the guide for scientists. In this PR I am also adding a "data" page.

### CODE REVIEW

**Comment by rabernat** [comment]
This is ready to review if anyone is game...

File: docs/data.rst [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+Guide to Preparing Cloud-Optimized Data
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+How do you get data into the cloud in a format that is optimized for pangeo
+to read?
+This guide outlines our current practices for converting netCDF data to
+Zarr_ and placing it on the cloud.
+These recommendations may change as cloud storage technology evolves.
+
+.. note::
```
Comment (dopplershift) [inline docs/data.rst]: This is rendering as plain text on GitHub, not sure if it's working properly when using Sphinx.

File: docs/data.rst [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+Google Cloud Storage Data Catalog
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Thanks to our :ref:`NSF Award <nsf-award>`, Pangeo has a substantial allocation
+on Google Cloud Platform which we are currently using to host several large
+datasets.
+These datasets are directly accessible from the Google Cloud Pangeo deployment:
+`pangeo.pydata.org <http://pangeo.pydata.org>`_.
+
+**HOW DO WE LIST THE DATASETS? CAN WE DIRECTLY PARSE THE INTAKE CATALOG?**
```
Comment (dopplershift) [inline docs/data.rst]: Do you want to commit with this in?

File: docs/data.rst (hunk 1 above)
Comment (rabernat) [reply docs/data.rst]: All looks good when I build locally.

File: docs/data.rst (hunk 2 above)
Comment (rabernat) [reply docs/data.rst]: No not really. We need to solve this. 😐

**Comment by rabernat** [comment]
I merged this for now... The site is still not really launched so its ok.

**Comment by vargava** [comment]
Hey y'all, I just noticed the following slip-by in this PR:
> Specific use case examples which load data from HPC platforms include:
> - Sean’s thing
> - What else?

It's live on the documentation and could be edited to showcase real examples. Thanks!

**Pull request merged**